recipes/flask-mail/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{%set name = "flask-mail" %}
+{%set camelName = "Flask-Mail" %}
```

Comment:
Ah sorry that is a bit vague. Basically was meaning one could do something like what is done in PR ( https://github.com/conda-forge/staged-recipes/pull/1415 ). In any event, this is just a suggestion. The effect is nearly identical.